.azdo/pipelines/azure-pipelines-admin.yml

```
@@ -195,7 +195,7 @@ stages:
           ContainerImage: ${{ variables.Container }}
 
   - stage: vNextE2eDeployApim
-    dependsOn:
+    dependsOn: 
```

Please remove whitespace added to end of line. No point in having more changed files than needed in PR! Thanks.
```suggestion
    dependsOn:
```

---

Removed extra space.